dart/dynamics/SoftBodyNode.h

```diff
@@ -361,7 +356,7 @@ class SoftBodyNode : public BodyNode
   UniqueProperties mSoftP;
 
   /// \brief Soft mesh shape belonging to this node.
-  std::shared_ptr<SoftMeshShape> mSoftShape;
+  ShapeNode* mSoftShapeNode;
```

Comment:
:+1: I guess you meant `TemplateNodePtr<ShapeNode>` or its typedef `ShapeNodePtr`.

Yeah, one day we need refactoring of soft body dynamics including Soft Shapes.